components/compliance-service/reporting/relaxting/profiles.go

```diff
@@ -146,12 +146,12 @@ func (esprofile *ESInspecProfile) parseInspecProfile(profile inspec.Profile) err
 		}
 		var refs string
 		byteRefs, _ := json.Marshal(control.Refs)
-		json.Unmarshal(byteRefs, refs) // nolint: errcheck
+		json.Unmarshal(byteRefs, &refs) // nolint: errcheck
```

Comment:
All these nolint: errcheck's make me nervous but it is existing code so I think we should leave it.

Reply:
I ran into trouble trying to sort this out before: https://github.com/chef/a2/pull/3158